### PR TITLE
Spatial_oM: Adding project

### LINKED ISSUE #806: Spatial_oM: Add Project
#### Description:

Add Spatial_oM to BHoM

### PR DESCRIPTION
**Pull request by IsakNaslundBh**
### Issues addressed by this PR

Closes #806

Setting up new Spatial_oM

### CODE REVIEW

File: Spatial_oM/Spatial_oM.csproj [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+    <Optimize>false</Optimize>
+    <OutputPath>..\Build\</OutputPath>
+    <DefineConstants>DEBUG;TRACE</DefineConstants>
+    <ErrorReport>prompt</ErrorReport>
+    <WarningLevel>4</WarningLevel>
+  </PropertyGroup>
+  <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Release|AnyCPU' ">
+    <DebugType>pdbonly</DebugType>
+    <Optimize>true</Optimize>
+    <OutputPath>bin\Release\</OutputPath>
```
Comment (FraserGreenroyd) [inline Spatial_oM/Spatial_oM.csproj:27]: Should be `..\Build\`

File: Spatial_oM/Spatial_oM.csproj [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+  <Import Project="$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props" Condition="Exists('$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props')" />
+  <PropertyGroup>
+    <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
+    <Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>
+    <ProjectGuid>{789EAF6C-D68F-4770-B01B-C07BBF3D2188}</ProjectGuid>
+    <OutputType>Library</OutputType>
+    <AppDesignerFolder>Properties</AppDesignerFolder>
+    <RootNamespace>BH.oM.Spatial</RootNamespace>
+    <AssemblyName>Spatial_oM</AssemblyName>
+    <TargetFrameworkVersion>v4.5</TargetFrameworkVersion>
```
Comment (kThorsager) [inline Spatial_oM/Spatial_oM.csproj:12]:
```suggestion
    <TargetFrameworkVersion>v4.5.2</TargetFrameworkVersion>
```

File: BHoM.sln [hunk 3]
```diff
@@ -1,7 +1,7 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio Version 16
-VisualStudioVersion = 16.0.29613.14
+# Visual Studio Express 14 for Windows Desktop
+VisualStudioVersion = 14.0.25420.1
```
Comment (kThorsager) [inline BHoM.sln:4]: Does this matter?

File: Spatial_oM/Spatial_oM.csproj (hunk 1 above)
Comment (IsakNaslundBh) [reply Spatial_oM/Spatial_oM.csproj:27]: ah, only changed the debug 🤦

File: BHoM.sln (hunk 3 above)
Comment (IsakNaslundBh) [reply BHoM.sln:4]: Should not really, just depends on which visual studio was used to build the solution the last time the solution file was changed.

Comment (FraserGreenroyd) [reply BHoM.sln:4]: It's not ideal, but it's accepted in most toolkits so should be ok.

**Comment by IsakNaslundBh** [comment]
🎈

**Pull request merged**